test/functional/tests/io_class/test_io_class_core_id.py

```diff
@@ -66,33 +62,19 @@ def test_ioclass_core_id(filesystem):
             cache_id=cache.cache_id, file=ioclass_config.default_config_file_path
         )
 
-    if filesystem:
-        with TestRun.step(f"Mount cores"):
-            core_1.mount(cached_mountpoint)
-            core_2.mount(not_cached_mountpoint)
-
     with TestRun.step(f"Reset counters"):
         sync()
         drop_caches()
         cache.purge_cache()
         cache.reset_counters()
 
     with TestRun.step(f"Trigger IO to both cores"):
-        if filesystem:
-            dd_dst_paths = [cached_mountpoint + "/test_file", not_cached_mountpoint + "/test_file"]
-        else:
-            dd_dst_paths = [core_1.path, core_2.path]
+        dd_dst_paths = [core_1.path, core_2.path]
 
         for path in dd_dst_paths:
-            dd = (
-                Dd()
-                .input("/dev/zero")
-                .output(path)
-                .count(dd_count)
-                .block_size(dd_bs)
-                .oflag("sync")
-            )
-            dd.run()
+            Dd().input("/dev/zero").output(path).count(dd_count).block_size(
+                dd_bs
+            ).oflag("sync").run()
```

Comment:
This is oddly formatted. I suppose "black" did that. ;)
Although it will work, it looks strange and it's hard to read.

Reply:
I've seen this construction also in a few other places here.